### PR TITLE
Reduce VPA mutating webhook timeout from 30s to 10s

### PR DESCRIPTION
**Pull request by dharmab**
Fixes https://github.com/kubernetes/autoscaler/issues/2762

### CODE REVIEW

**Comment by dharmab** [comment]
A follow-up PR could make this timeout configurable, but reducing the timeout from the default 30s to the much saner 10s is an improvement and reflects the change in default between the admissionregistration.k8s.io/v1beta1 and admissionregistration.k8s.io/v1 types.

File: vertical-pod-autoscaler/pkg/admission-controller/config.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ func selfRegistration(clientset *kubernetes.Clientset, caCert []byte, namespace,
 		RegisterClientConfig.URL = &url
 	}
 	sideEffects := v1beta1.SideEffectClassNone
+	timeoutSeconds := int32(10)
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/admission-controller/config.go]: Can we make it a flag?

**Comment by dharmab** [comment]
I've decided to close this in favor of using https://github.com/kubernetes/autoscaler/pull/2949 instead.

**Pull request closed without merging**